scripts/zones/Davoi/mobs/Three-eyed_Prozpuz.lua

```diff
@@ -0,0 +1,21 @@
+-----------------------------------
+-- Area: Davoi
+-- NPC: Three-Eyed Prozpuz
```

Comment:
same here, script does nothing, don't need it. The mob will still pop without this empty script.